### PR TITLE
Update Interaction plots for v15

### PR DESCRIPTION
**Pull request by jashapiro**
#### What scientific question is your analysis addressing?

Updating analysis for v15

#### What was your approach?

Replaced `disease-type-new` with `integrated-diagnosis` and reran on full data
Changes in subset used for CI were also required due to changes in the included subset.

#### What GitHub issue does your pull request address?

#574

#### Which areas should receive a particularly close look?

CI config needs to be reverted after passing, and will not pass at that point

#### Reproducibility Checklist

- [ ] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [ ] This analysis has been added to continuous integration.

#### Documentation Checklist

- [ ] This analysis module has a `README` and it is up to date.
- [ ] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [ ] The analytical code is documented and contains comments.

### CODE REVIEW

**Comment by jashapiro** [comment]
Passed CI with only this module running.

File: analyses/interaction-plots/scripts/01-disease-specimen-lists.R [hunk 1]
```diff
@@ -85,16 +85,15 @@ out_file <- file.path(root_dir, opts$outfile)
 
 #### Read files
 
-meta_df <- readr::read_tsv(meta_file, col_types = readr::cols())
+meta_df <- readr::read_tsv(meta_file, col_types = readr::cols(), guess_max = 10000)
```
Comment (cbethell) [inline analyses/interaction-plots/scripts/01-disease-specimen-lists.R]: What is the reason behind adding the `guess_max` argument to the function here?

Comment (jashapiro) [reply analyses/interaction-plots/scripts/01-disease-specimen-lists.R]: It prevents a warning when readr guesses the wrong data type for some columns (columns that are blank for the first 1000 samples, in this case)

**Comment by jashapiro** [comment]
Merging over CI failure (for other modules)

**Pull request merged**